<ui version="4.0" >
 <class>VP12DExaminerViewerForm</class>
 <widget class="QWidget" name="VP12DExaminerViewerForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>VP12DExaminerViewer</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icons.qrc" >:/icons/icons/view_all.png</iconset>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" native="1" name="widget_viewchangebuttons" />
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_views" >
     <property name="minimumSize" >
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <widget class="QWidget" name="page" />
     <widget class="QWidget" name="page_2" />
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <property name="horizontalSpacing" >
        <number>0</number>
       </property>
       <property name="verticalSpacing" >
        <number>0</number>
       </property>
       <item row="3" column="0" >
        <widget class="QPushButton" name="pushButton_sethome" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/set_home.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0" >
        <widget class="QPushButton" name="pushButton_zoom" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/crystalicon_zoom.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="pushButton_changeview" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/view.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0" >
        <widget class="QPushButton" name="pushButton_zoomminus" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/crystalicon_zoom_editted_minus.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="pushButton_home" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/home.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QPushButton" name="pushButton_viewall" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/view_all.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QPushButton" name="pushButton_pick" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/pick.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0" >
        <widget class="QPushButton" name="pushButton_zoomplus" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/crystalicon_zoom_editted_plus.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QPushButton" name="pushButton_seek" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/seek.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QPushButton" name="pushButton_print" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="icons.qrc" >:/icons/icons/fileprint.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections/>
</ui>
